lms/djangoapps/teams/static/teams/js/spec/views/my_teams_spec.js

```diff
@@ -28,26 +30,52 @@ define([
         it('can render itself', function () {
             var teamMembershipsData = TeamSpecHelpers.createMockTeamMembershipsData(1, 5),
                 teamMemberships = TeamSpecHelpers.createMockTeamMemberships(teamMembershipsData),
-                teamsView = createMyTeamsView({
+                myTeamsView = createMyTeamsView({
                     teams: teamMemberships,
                     teamMemberships: teamMemberships
                 });
 
-            TeamSpecHelpers.verifyCards(teamsView, teamMembershipsData);
+            TeamSpecHelpers.verifyCards(myTeamsView, teamMembershipsData);
 
             // Verify that there is no header or footer
-            expect(teamsView.$('.teams-paging-header').text().trim()).toBe('');
-            expect(teamsView.$('.teams-paging-footer').text().trim()).toBe('');
+            expect(myTeamsView.$('.teams-paging-header').text().trim()).toBe('');
+            expect(myTeamsView.$('.teams-paging-footer').text().trim()).toBe('');
         });
 
         it('shows a message when the user is not a member of any teams', function () {
             var teamMemberships = TeamSpecHelpers.createMockTeamMemberships([]),
-                teamsView = createMyTeamsView({
+                myTeamsView = createMyTeamsView({
                     teams: teamMemberships,
                     teamMemberships: teamMemberships
                 });
-            TeamSpecHelpers.verifyCards(teamsView, []);
-            expect(teamsView.$el.text().trim()).toBe('You are not currently a member of any teams.');
+            TeamSpecHelpers.verifyCards(myTeamsView, []);
+            expect(myTeamsView.$el.text().trim()).toBe('You are not currently a member of any team.');
+        });
+
+        it('refreshes a stale membership collection when rendering', function() {
+            var requests = AjaxHelpers.requests(this),
+                teamMemberships = TeamSpecHelpers.createMockTeamMemberships([]),
+                myTeamsView = createMyTeamsView({
+                    teams: teamMemberships,
+                    teamMemberships: teamMemberships
+                }),
+                request, url, parameters;
+            TeamSpecHelpers.verifyCards(myTeamsView, []);
+            expect(myTeamsView.$el.text().trim()).toBe('You are not currently a member of any team.');
+            teamMemberships.teamEvents.trigger('teams:update', { action: 'create' });
```

Comment:
It does the same for add and join team, correct? Not that we necessarily need to test them, just want to make sure.